src/docs/development/tools/pubspec.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Flutter and the pubspec file"
+description: "Describes the Flutter-only fields in the pubspec file."
+---
+
+Every Flutter project includes a `pubspec.yaml` file,
+often referred to as _the pubspec_.
+A basic pubspec is generated when you create
+a new Flutter project. It's located at the top
+of the project tree and contains metadata about
+the project that the Dart and Flutter tooling
+needs to know.
+
+The pubspec file specifies dependencies
+that the project requires, such as particular packages
```

Comment:
This page seems to mostly be relevant to apps, though there's some overlap.  Maybe move the `/docs/development/packages-and-plugins/developing-packages` link up to point out where to look for more info if they are writing packages and plugins?

Reply:
Great idea. Done!